apps/codeforafrica/src/components/AddressCard/AddressCard.js

```diff
@@ -38,12 +47,37 @@ const AddressCard = React.forwardRef(function AddressCard(props, ref) {
             {title}
           </RichTypography>
           <RichTypography
-            component="address"
             // In address, we treat <p> as a line i.e.no margins.
             sx={{ color: "inherit", pt: 1.25, "& p": { m: 0 } }}
             variant="body3"
           >
-            {content}
+            {addressLine1}
+          </RichTypography>
+          <RichTypography
+            sx={{ color: "inherit", "& p": { m: 0 } }}
+            variant="body3"
+          >
+            {addressLine2}
+          </RichTypography>
+          <RichTypography
+            sx={{ color: "inherit", "& p": { m: 0 } }}
+            variant="body3"
+            component="span"
+          >
+            {zipcode},
```

Comment:
Only include `,` if there is `zipCode`. From collection definition, `zipCode` is not required and hence may be empty.